charts/ckan/values.schema.json

```diff
@@ -0,0 +1,259 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema",
+  "type": "object",
+  "title": "ckan configuration",
+  "properties": {
+    "global": {
+      "type": "object",
+      "properties": {
+        "imageRegistry": {
+          "type": "string",
+          "description": "The global container image proxy, e.g. [Nexus](https://artifacthub.io/packages/helm/sonatype/nexus-repository-manager), this needs to support various registries",
+          "examples": [
+            "nexus.teuto.net"
+          ]
+        },
+        "imagePullSecrets": {
+          "type": "array"
+        },
+        "storageClass": {
+          "$ref": "#/$defs/storageClass"
+        }
+      },
+      "ckan": {
+        "type": "object",
+        "properties": {
+          "replicas": {
+            "type": "integer"
+          },
+          "siteId":{
+            "type": "string",
+            "decription": ""
+          },
+          "siteTitle": {
+            "type": "string",
+            "description": ""
+          },
+          "image": {
+            "$ref": "#/$defs/image"
+          },
+          "ingress": {
+            "type": "object",
+            "properties": {
+              "enables": {
+                "type": "boolean"
+              },
+              "className": {
+                "type": "string"
+              },
+              "annotations": {
+                "type": "object"
+              },
+              "baseDomain": {
+                "type": "string",
+                "description": "The base domain to be used for cluster ingress",
+                "examples": [
+                  "teuto.net"
+                ]
+              }
```

Comment:
This is never _used_ as a real base domain, why not `host`?

Especially since this in the `ckan` block, so explicitly *not* a baseDomain

And the description doesn't match 😉